build-tools/packages/build-cli/src/library/commands/generateEntrypoints.ts

```diff
@@ -362,6 +362,9 @@ async function generateEntrypoints(
 	const mainSourceFile = project.addSourceFileAtPath(mainEntrypoint);
 	const exports = getApiExports(mainSourceFile);
 
+	const packageDocumentationHeader = getPackageDocumentationText(mainSourceFile);
+	const newFileHeader = `${generatedHeader}${packageDocumentationHeader}`;
```

Comment:
Presumably TSMorph preserves line breaks, but omits the leading `* ` of each line? So just concatenating like this is safe?

A unit test would be nice for illustrating 😜

Reply:
TSMorph preserves the whitespace after in the testing I did. The `generatedHeader` has trailing lines.
Example result:
```typescript
/*!
 * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
 * Licensed under the MIT License.
 */

/*
 * THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
 * Generated by "flub generate entrypoints" in @fluidframework/build-tools.
 */

/**
 * This library contains the interfaces and types concerning the `Loader` and loading the `Container`.
 *
 * @packageDocumentation
 */

export {
```

Agree on the test debt.